sdk/download.go

```diff
@@ -318,3 +319,18 @@ func UpdateSignedFile(file, url string, client *http.Client) error {
 	plog.Infof("Verified file: %s", file)
 	return nil
 }
+
+func DetectSDKVersion(url, branch string) (string, error) {
+	tmp, err := ioutil.TempDir("", "")
+	if err != nil {
+		return "", err
+	}
+	defer os.RemoveAll(tmp)
+
+	clone := exec.Command("git", "clone", "--depth=1", "--single-branch", "-b", branch, url, tmp)
```

Comment:
Ah, and git doesn't have a quiet option? Perhaps then use CombinedOutput and dump on error.